e2e/cypress/integration/channel/more_channels_spec.js

```diff
@@ -212,6 +212,7 @@ describe('Channels', () => {
                 // * Modal should show the archived channels list
                 cy.wrap(el).should('contain', 'Show: Archived Channels');
             }).wait(TIMEOUTS.HALF_SEC);
+            cy.get('#searchChannelsTextbox').clear();
             cy.get('#moreChannelsList').should('be.visible').children().should('have.length', 2);
```

Comment:
Failed on me here. Maybe this should be tested in a new team, instead of shared `testTeam`.
![Screen Shot 2021-03-11 at 7 21 20 PM](https://user-images.githubusercontent.com/5334504/110780754-011d2e80-82a0-11eb-979e-2ad5c655b91c.png)